forms/mainwindow.ui

```diff
@@ -3022,6 +3023,11 @@
     <string>Export Map Stitch Image...</string>
    </property>
   </action>
```

Comment:
Maybe we should add another separator?
![image](https://user-images.githubusercontent.com/2229288/94215793-40e38b80-fea3-11ea-8842-6cdd4ac23b11.png)

Reply:
I'd expect New Map and New Tileset in the File menu, even though they bring up option dialogs rather than file choosers.